### DISCUSSION TITLE
PhysX Collision Algorithms: ABP and Persistent Contact Manifold

### DISCUSSION THREAD

**Discussion by oneanime** [NVIDIA-Omniverse/PhysX#332] [Q&A]
Hello, I am currently studying the collision algorithms in PhysX, specifically the broad-phase algorithm 'Automatic Box Pruning (ABP)' and the narrow-phase algorithm 'Persistent Contact Manifold'. Could you recommend any papers on these topics?

**Comment by foured** [comment]
Hi! Did you find anything good?
